scripts/feature_tests/upgrade/upgrade_vars.sh

```diff
@@ -6,11 +6,17 @@ set -eux
 
 export CAPIRELEASE_HARDCODED="v1.0.99"
 
-export CAPIRELEASE="v1.0.0"
+export CAPI_VERSION="v1alpha4"
+export CAPIRELEASE="v0.4.5"
 export CAPI_REL_TO_VERSION="v1.0.1"
 
-export CAPM3RELEASE="v1.0.0"
+export CAPM3_VERSION="v1alpha5"
+export CAPM3RELEASE="v0.5.3"
 export CAPM3_REL_TO_VERSION="v1.0.0"
+export UPGRADED_CAPM3_VERSION="v1beta1"
+
+# Ubuntu is hard coded in the upgrade tests. Make sure we use it throughout.
+export IMAGE_OS="Ubuntu"
```

Comment:
Hard-coded? Why do we need this value to be set here? Shouldn't it be coming through jjb?

Reply:
Yes jjb is setting it correctly, I was just thinking that we may as well hard code `IMAGE_OS` for upgrade tests since it is [hard coded here](https://github.com/metal3-io/metal3-dev-env/blob/620b4e6abef8c0edaedd8dc3cdab2a5b3d0da7bd/vm-setup/roles/v1aX_integration_test/tasks/upgrade.yml#L334). For the jjb/Jenkins jobs I guess it doesn't matter, but if you want to run them manually it will break if you forget to set this.

Reply:
Aha , makes sense now.